src/reference/00-Getting-Started/06-Scopes.md

```diff
@@ -24,7 +24,7 @@ context, called a "scope."
 
 Some concrete examples:
 
-- if you have multiple projects in your build definition, a key can
+- if you have multiple projects (also called subprojects) in your build definition, a key can
   have a different value in each project.
 - the `compile` key may have a different value for your main sources and
   your test sources, if you want to compile them differently.
```

Comment:
A code example up here somewhere would be useful. I read through pretty much the whole page here wondering how I specified a scope. Some of the content in the "Referring to scopes in a build definition" section would be great to move up towards the top.